pkg/cache/node_info.go

```diff
@@ -56,6 +58,7 @@ func NewNodeInfo(proto *si.NewNodeInfo) *NodeInfo {
 		NodeID:            proto.NodeID,
 		totalResource:     resources.NewResourceFromProto(proto.SchedulableResource),
 		allocatedResource: resources.NewResource(),
+		occupiedResource:  resources.NewResource(),
```

Comment:
Are you sure that this is always zero when we register a new node or could we have an allocation already?

Reply:
The existing allocations for non-yunikorn-scheduled pods will be reported through UPDATE call. Not part of existing allocations.

Reply:
perfect that is what I wanted to know